test_config.py

```diff
@@ -1,5 +1,5 @@
 import sys
-
+from benchopt.utils.sys_info import _get_cuda_version
 import pytest
```

Comment:
Put `benchopt` related import at the end of the imports (from more general stdlib to particular packages)

```suggestion
import pytest

from benchopt.utils.sys_info import _get_cuda_version
```